pkg/branding/brandedtext_test.go

```diff
@@ -0,0 +1,37 @@
+package branding
+
+import (
+	"testing"
+
+	"github.com/stackrox/rox/pkg/testutils/envisolator"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetBrandedProductName(t *testing.T) {
+	envIsolator := envisolator.NewEnvIsolator(t)
+
+	tests := map[string]struct {
+		productBrandingEnv string
+		brandedProductName string
+	}{
+		"RHACS branding": {
+			productBrandingEnv: "RHACS_BRANDING",
+			brandedProductName: productBrandingNameRHACS,
+		},
+		"Stackrox branding": {
+			productBrandingEnv: "STACKROX_BRANDING",
+			brandedProductName: productBrandingNameStackrox,
+		},
+		"Default setting": {
+			productBrandingEnv: "ROX_PRODUCT_BRANDING",
+			brandedProductName: productBrandingNameStackrox,
+		},
```

Comment:
Quite unusual, it appears to be similar to
```bash
export ROX_PRODUCT_BRANDING="ROX_PRODUCT_BRANDING"
```
Is it really what you wanted to mean here?
I think a test that demonstrates that any _unrecognized_ value leads to open source branding is good and useful, I just think that the `"Default setting"` name is confusing.